checks/branch_protected.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const branchProtectionStr = "Branch-Protection"
+const minReviews = 2
```

Comment:
i dont think we are ready for 2 mandatory review stick, probably lets keep 1 for now ? no project enforces this, so we will see all failures, maybe this 2 policy is better for Jeff's work where we can create org wide policy for stuff.

Reply:
sg.